docs/using_zampy.md

```diff
@@ -13,26 +13,28 @@ This file is created under your -*user's home*-/.config directory:
 
 `~/.config/zampy/zampy_config.yml`
```

Comment:
Is there a reason for using `yml` extension instead of `yaml`?

Reply:
`yml` is more commonly used, but apparently `yaml` is the "officially prefered way". I don't really care.

Reply:
Also, the github workflows are .yml by default, conda envs are .yml by default, and ESMValTool uses .yml.